deployment/v3/utils/prop_migrator/knowledge/old-value-takes-priority.csv

```diff
@@ -46,88 +46,14 @@ registration-processor-default.properties,packetmanager.provider.uingenerator.pa
 registration-processor-default.properties,packetmanager.provider.uingenerator.parentOrGuardianRID
 registration-processor-default.properties,registration.processor.reprocess.elapse.time
 registration-processor-default.properties,registration.processor.reprocess.hours
-registration-processor-default.properties,mosip.registration.processor.database.port
 registration-processor-default.properties,registration.processor.LANDING_ZONE
 registration-processor-default.properties,mosip.regproc.packet.classifier.tagging.agegroup.ranges
-registration-processor-default.properties,mosip.kernel.virus-scanner.host
-registration-processor-default.properties,registration.processor.reprocess.minutes:
-registration-processor-default.properties,mosip.kernel.virus-scanner.port:
-registration-processor-default.properties,registration.processor.queue.username:
-registration-default.properties,mosip.registration.num_of_fingerprint_retries
-registration-default.properties,mosip.registration.num_of_iris_retries
-registration-default.properties,mosip.registration.num_of_face_retries
-registration-default.properties,mosip.registration.leftslap_fingerprint_threshold
-registration-default.properties,mosip.registration.rightslap_fingerprint_threshold
-registration-default.properties,mosip.registration.thumbs_fingerprint_threshold
-registration-default.properties,mosip.registration.iris_threshold
```

Comment:
this have only 5 entries removed

Reply:
apart from yes and yes-vish we also removed the #N/A properties aswell sir that's why the count is high.

Reply:
in old value list, NA has only 2 records, also we din't discuss to remove NA for old value takes priority

Reply:
as per the discussion we have removed the duplicates which are present in both latest as well as old takes priority.

Reply:
No we have not discussed that, we have to remove NA only in the lastest takes priority, items in the old one should be retained. Also the count is not matching.
Items marked for delete using yes, yes-vish is only 5 items.
Items marked as NA is only 2 items in the old takes priority list.
But lines deleted is much more than the combined count which is 7